diff --git a/node_modules/@solana/wallet-adapter-react-ui/lib/esm/WalletListItem.js b/node_modules/@solana/wallet-adapter-react-ui/lib/esm/WalletListItem.js
index a0b24c2..dd61bd3 100644
--- a/node_modules/@solana/wallet-adapter-react-ui/lib/esm/WalletListItem.js
+++ b/node_modules/@solana/wallet-adapter-react-ui/lib/esm/WalletListItem.js
@@ -3,9 +3,11 @@ import React from 'react';
 import { Button } from './Button.js';
 import { WalletIcon } from './WalletIcon.js';
 export const WalletListItem = ({ handleClick, tabIndex, wallet }) => {
+    const availableWallets = ["Phantom", "Salmon", "Backpack", "MetaMask"]
+    const isWalletAvailable = availableWallets.includes(wallet.adapter.name)
     return (React.createElement("li", null,
-        React.createElement(Button, { onClick: handleClick, startIcon: React.createElement(WalletIcon, { wallet: wallet }), tabIndex: tabIndex },
+        React.createElement(Button, { onClick: handleClick, startIcon: React.createElement(WalletIcon, { wallet: wallet }), tabIndex: tabIndex, disabled:!isWalletAvailable },
             wallet.adapter.name,
-            wallet.readyState === WalletReadyState.Installed && React.createElement("span", null, "Detected"))));
+            wallet.readyState === WalletReadyState.Installed && isWalletAvailable ? React.createElement("span", null, "Detected") : React.createElement("span", null, "Not supported"))));
 };
 //# sourceMappingURL=WalletListItem.js.map
\ No newline at end of file
